src/integration/create-express-middleware.js

```diff
@@ -0,0 +1,262 @@
+import express from 'express';
+import request from 'supertest';
+import { createExpressMiddleware } from '../create-express-middleware';
+import { describe } from 'riteway';
+
+describe('createExpressMiddleware()', async should => {
+  const { assert } = should();
+  {
+    const app = express();
+    const initialFeatures = [
```

Comment:
We don't need all these features to test this :D 

We just need enough to verify that the middleware handler is doing what its supposed to do.

Reply:
For context: The only reason these tests originally used that many features is because the fixture provided them all for another test that needed to test feature dependencies

Reply:
Ok. I was probably just being too lazy and logged out the values from the previous tests to use them. Shortened it up now though.